api-specificatie/BRP-Wijzigingen/openapi.yaml

```diff
@@ -0,0 +1,146 @@
+openapi: 3.0.0
+info:
+  title: "BRP Wijzigingen API"
+  version: "1.0.0"
+  contact:
+    name: test
+    email: me@home.nl
+    url: http://home.nl
+paths:
+  /wijzigingen:
+    get:
+      tags: 
+      - wijzigingen
+      operationId: getWijzigingen
+      parameters:
+        - name: afnemersindicatie
+          in: query
+          description: afnemersindicatie
+          required: true
+          schema:
+            type: string
+        - name: van
+          in: query
+          description: "van (yyyy-mm-ddThh:mm:ss)"
```

Comment:
Moet je de hele tijdstip opgeven? Of kan je ook een datum opgeven?

Reply:
Ik denk dat dit afhankelijk is van het aantal keren per dag dat wijzigingen opgevraagd worden. Soms is 1x per dag voldoende maar in andere gevallen moeten deze vaker opgevraagd worden.
Ik kan uitzoeken of het mogelijk is om zowel een formaat met tijdstip als een formaat zonder tijdstip te ondersteunen. Ik vraag me wel af of het erg is dat, als je slechts 1x per dag een wijziging opvraagt, je naast de datum ook in een tijdstip moet voorzien.